### PR TITLE
Added FAQ

### PR DESCRIPTION
**Pull request by tanmayanand44**
Done #137

### CODE REVIEW

File: src/pages/FAQ.js [hunk 1]
```diff
@@ -65,6 +65,25 @@ export default class FAQ extends Component {
               Once a sufficient percentage of DID holders have approved a pull
               request, DID are automatically issued on the blockchain.
             </p>
+            <p style={{ fontSize: '1.33em', marginTop: '2.2em' }}>
+              <Header as="h3">What id DID?</Header>
```
Comment (JohnAllen) [inline src/pages/FAQ.js]: Let's fix this typo here @tanmayanand44 "What are DID?"

File: src/pages/FAQ.js [hunk 2]
```diff
@@ -65,6 +65,25 @@ export default class FAQ extends Component {
...
+            </p>
+            <p style={{ fontSize: '1.10em' }}>
+              1.)DID are our economic and {'\n'}governance token
```
Comment (JohnAllen) [inline src/pages/FAQ.js]: Let's also remove the numbers and use a _bulleted_ Semantic-UI [list](https://semantic-ui.com/elements/list.html#bulleted)

**Comment by JohnAllen** [comment]
@tanmayanand44 Want me to close this?

**Comment by tanmayanand44** [comment]
@JohnAllen You can close.I will be adding with bulleted list soon.

**Pull request closed without merging**